SAMLPortal/Models/SAMLPortalContext.cs

```diff
@@ -29,6 +30,16 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 		protected override void OnModelCreating(ModelBuilder modelBuilder)
 		{
 			OnModelCreatingPartial(modelBuilder);
+
+			// Will come later when adding multiple role to one app
+			// var stringArrayConverter = new ValueConverter<string[], string>(
+			// v => string.Join(";", v),
+			// v => v.Split(";", StringSplitOptions.RemoveEmptyEntries));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5434886)